tutorials/darcy_thermo_mech/doc/content/workshop/systems/outputs.md

```diff
@@ -88,8 +90,6 @@ using the short-cut syntax.  sub-blocks use the actual sub-block name as the suf
 []
 ```
 
-The use of 'file_base' anywhere in the `[Outputs]` block disables all default naming behavior.
```

Comment:
I tested this. This statement is false. The default naming behavior is only disabled in the block where `file_base` is specified.